samples/benchmarks/coremark/Kconfig.sysbuild

```diff
@@ -20,6 +20,6 @@ config APP_CPUPPR_RUN
 	  benchmark only on two cores at a time; application core and either radio core or PPR
 	  core.
 
-	  PPR code is run from MRAM with the execution in place (XIP) method. You must use the
-	  "nordic-ppr-xip" snippet for the application core to be able to boot the PPR core.
-	  You need to provide the "coremark_SNIPPET=nordic-ppr-xip" build argument.
+	  PPR code is run from RAM. You must use the "nordic-ppr" snippet for the application
```

Comment:
Maybe lets inform exactly that this is tightly couple memory for ppr - ram RAM30 (TCM for PPR).

Reply:
We did not provide such information for MRAM in the XIP mode (the exact MRAM block number that is used for the specific core). Getting into the platform-specific details here would make this documentation harder to maintain